scripts/qa_crowbarsetup.sh

```diff
@@ -1465,7 +1465,12 @@ EOF
     local netfile="/etc/crowbar/network.json"
 
     local netfilepatch=`basename $netfile`.patch
-    [ -e ~/$netfilepatch ] && patch -p1 $netfile < ~/$netfilepatch
+    if [ -e ~/$netfilepatch ]; then
+        if ! which patch 2>&1 >/dev/null; then
+            safely zypper --no-gpg-checks -n in patch
```

Comment:
you could even get rid of the `which patch` then, because it checks the rpm DB already, which is fast.